docs/source/openmw-cs/tour.rst

```diff
@@ -13,7 +13,7 @@ installed and playable.
 Adding the ring to the game's records
 *************************************
 
-In this first section we will define what our new ring is, what it looks like
+In this first section we will define what our new ring is, what it looks like,
```

Comment:
The Oxford comma is dead.

 http://www.adweek.com/galleycat/oxford-comma-dropped-by-university-of-oxford/34361?red=as